media-libs/opencolorio/patches/opencolorio-2.3.2.patchset

```diff
@@ -0,0 +1,31 @@
+From 2521e7480806b0b374318ef11cfff2ea3772236e Mon Sep 17 00:00:00 2001
+From: Schrijvers Luc <begasus@gmail.com>
+Date: Tue, 18 Jun 2024 19:06:09 +0200
+Subject: Fix build (hackish)
+
+
+diff --git a/src/utils/NumberUtils.h b/src/utils/NumberUtils.h
+index 08a9ba0..8b0ec8d 100644
+--- a/src/utils/NumberUtils.h
++++ b/src/utils/NumberUtils.h
+@@ -65,7 +65,7 @@ really_inline from_chars_result from_chars(const char *first, const char *last,
+ #ifdef _WIN32
+     tempval = _strtod_l(first, &endptr, loc.local);
+ #else
+-    tempval = ::strtod_l(first, &endptr, loc.local);
++    tempval = ::strtod(first, &endptr);
+ #endif
+ 
+     if (errno != 0 && errno != EINVAL)
+@@ -109,7 +109,7 @@ really_inline from_chars_result from_chars(const char *first, const char *last,
+     // On OSX, strtod_l is for some reason drastically faster than strtof_l.
+     tempval = static_cast<float>(::strtod_l(first, &endptr, loc.local));
+ #else
+-    tempval = ::strtof_l(first, &endptr, loc.local);
++    tempval = ::strtod(first, &endptr);
```

Comment:
I think this should be `::strtof` (from the context).

Reply:
Good call, will check!

Reply:
Fixed. Thanks!